src/preset_cli/cli/superset/export.py

```diff
@@ -77,18 +77,29 @@ def export_assets(  # pylint: disable=too-many-locals, too-many-arguments
         "chart": {int(id_) for id_ in chart_ids},
         "dashboard": {int(id_) for id_ in dashboard_ids},
     }
+    ids_requested = database_ids or dataset_ids or chart_ids or dashboard_ids
```

Comment:
NIT(optional): could be a tiny bit more clear is 
```python
were_any_ids_requested = any(database_ids, dataset_ids, chart_ids, dashboard_ids)
```

Reply:
Ah, I like it! Let me fix.